data/strings/strings.txt

```diff
@@ -30173,3 +30173,94 @@
     vi = Thu nhỏ
     zh-Hans = 放大
     zh-Hant = 縮小
+
+  [website_menu]
+    comment = Text in menu
+    tags = android
+    en = Menu link
+    af = Kieslys skakel
+    ar = رابط القائمة
+    az = Menyu keçidi
+    be = Спасылка на меню
+    bg = Връзка към менюто
+    ca = Enllaç del menú
+    cs = Odkaz na menu
+    da = Link til menu
+    de = Link zur Speisekarte
+    el = Σύνδεσμος μενού
+    es = Enlace al menú
+    et = Menüü link
+    eu = Menuaren esteka
+    fa = لینک منو
+    fi = Valikkolinkki
+    fr = Lien vers le menu
+    he = קישור לתפריט
+    hi = मेनू लिंक
+    hu = Menü link
+    id = Tautan menu
+    it = Link al menu
+    ja = メニューリンク
+    ko = 메뉴 링크
+    lt = Meniu nuoroda
+    mr = मेनू लिंक
+    nb = Lenke til menyen
+    nl = Menulink
+    pl = Link do menu
+    pt = Link do menu
+    ro = Legătura de meniu
+    ru = Ссылка на меню
+    sk = Odkaz na menu
+    sv = Länk till meny
+    sw = Kiungo cha menyu
+    th = ลิงค์เมนู
+    tr = Menü bağlantısı
+    uk = Посилання на меню
+    vi = Liên kết thực đơn
+    zh-Hans = 菜单链接
+    zh-Hant = 選單連結
+
+  [view_menu]
+    comment = Text in menu
+    tags = android
+    en = View Menu
+    af = Bekyk spyskaart
+    ar = عرض القائمة
+    az = Menyuya baxın
+    be = Прагляд меню
+    bg = Преглед на менюто
+    ca = Veure menú
+    cs = Zobrazit menu
+    da = Vis menu
+    de = Speisekarte ansehen
```

Comment:
```suggestion
    de = Speisekarte
```

In German, there is a separate word for a restaurant menu. Other languages need to distinguish a restaurant menu vs application menu.